tests/_sector/referral/slugs_rewards_test.sql

```diff
@@ -0,0 +1,16 @@
+WITH unit_test AS (
+    SELECT
+        test.reward_amount_raw = actual.reward_amount_raw AS reward_amount_test,
+        test.is_referral = actual.is_referral AS is_referral_test
```

Comment:
does `actual.is_referral` exist in the spell? looks like the test is failing here https://github.com/duneanalytics/spellbook/actions/runs/6854507633/job/18637614208?pr=4777

Reply:
Ah no it doesn't, silly mistake by me. It does exist in the 'final' spell but not in the slugs model, will fix